docs/samples/browser-plugin-meetings/app.js

```diff
@@ -1554,6 +1554,23 @@ function createStageLayoutVideos() {
   document.getElementById('multistream-videos-container').appendChild(thumbnailsFieldset);
 }
 
+function createScreenshareView() {
+  allVideoPanes.screenShare = [];
+  allVideoPanes.thumbnails = [];
+
+  // create the thumbnails
+  const thumbnailsContainer = createContainer();
+
+  for (let i = 0; i < 8; i += 1) {
```

Comment:
This is the number of panes in that screen share view layout. Ideally this should be equal to DefaultConfiguration.video.layouts.ScreenShareView.activeSpeakerVideoPaneGroups[0].numPanes but because the sample app is pure js we can't import the DefaultConfiguration from remoteMediaManager.ts. I'll define a constant and add this info in the comment.

Reply:
done